src/lib/cli/templates/rule-script-ts.hbs

```diff
@@ -70,8 +69,8 @@ const rule: IRuleBuilder = {
     },
     meta: {
         docs: {
-            category: '{{category}}',
-            description: '{{description}}'
+            category: `{{category}}`,
```

Comment:
@sarvaje @molant  The reason of using `escapeSafeString` function instead of triple curly brace is that the backticks (`) need to be escaped by adding a backslash, otherwise the following might be generated if only using the triple curly braces:
```
...
description: `This is a `important` rule.`
...
```